assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/clip/config.py

```diff
@@ -18,6 +18,7 @@ class Tasks(_CustomEnum):
     """Tasks supported."""
 
     ZERO_SHOT_IMAGE_CLASSIFICATION = "zero-shot-image-classification"
+    IMAGE_TEXT_EMBEDDINGS = "image-text-embeddings"
```

Comment:
Could you please check with Rashaud if this task name is okay ? It is better to use the same task name everywhere - UI filter, model wrapper etc.

Reply:
Followed up offline, we decided to use the task name "embeddings"